.github/workflows/bumper.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   update-dep:
     strategy:
       matrix:
-        deno: ["1.1.2"]
+        deno: ["1.1.3"]
```

Comment:
Should we do this? I mean, dmm wasn't released with this deno version in mind. But honestly, since new deno versions are not breaking dmm, we could just as well always target the latest deno version.

Reply:
Yeah I see what you mean actually - what would you prefer?

Reply:
In my opinion we should only change when dmm releases a new version. dmm itself can target master since if an error occurs, we just fix it for that version and we are good to go.
And maybe dmm is a good candidate to keep it's version in sync with deno, since we are not going to add that many features. That way we use the same variable to target dmm and deno.

Reply:
Makes sense, though I guess as we’re pointing to dmm master and dmm is running 1.1.3 atm we should be good on that front, but I can later tonight adjust the version dmm uses in its bumper?

Reply:
Honestly, this is a problem because `std` is still not v1... After it is, nothing should break.